90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -18,7 +18,7 @@ zlog() {
   [ "${1}" -le "${loglevel:=4}" ] || return
 
   # shellcheck disable=SC2086
-  _script="$( basename $0 )"
+  _script="$( basename -- $0 )"
```

Comment:
Is this going to break when we start using FreeBSD userland?

Reply:
Yes, horribly. Patches welcome.